src/controllers/messages/users/buyer.js

```diff
@@ -164,9 +165,40 @@ async function removeUserFromQueue(recipient, listingId, title) {
   }
 }
 
+function initializeQueueHandler(listingId) {
+  const queueRef = db.ref(`listings/${listingId}/queue`);
+  queueRef.on("value", async snapshot => {
+    const queue = snapshot.val();
+    console.log(queue);
+    if (queue) {
+      const firstInLine = queue[0];
+      const replies = [
+        {
+          content_type: "text",
+          title: "Yes",
+          payload: "accept-seller-offer"
+        },
+        {
+          content_type: "text",
+          title: "No",
+          payload: "decline-seller-offer"
+        }
+      ];
+      setContext(firstInLine, "accept-price", { listingId });
```

Comment:
Should I replace this with the state transition thing once I integrate it?

Reply:
Yeah I think so. Although I want to get the features working first before making that refactor.